### PR TITLE
infra: bump patch filter to 1.4

### PR DESCRIPTION
**Pull request by romani**
Attempt to fix execution on windows

### CODE REVIEW

**Comment by romani** [comment]
@Bananeweizen , please recheck if it works or does not make it worse

**Comment by Bananeweizen** [comment]
Works great. I've tested with the same change as before and get the previously missing violation CustomDeclarationOrder.

**Review by Bananeweizen — APPROVED** [review]

**Comment by romani** [comment]
@rnveach , FYI.

**Comment by romani** [comment]
@Bananeweizen , please do not hesitate to share any feedback on this filter.

File: pom.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
         <!-- sevntu and patch-filters need to use a compatible/compiled version with checkstyle -->
         <maven.sevntu.checkstyle.plugin.checkstyle.version>10.0</maven.sevntu.checkstyle.plugin.checkstyle.version>
         <maven.sevntu.checkstyle.plugin.version>1.42.0</maven.sevntu.checkstyle.plugin.version>
-        <maven.patch.filters.plugin.version>1.2.0</maven.patch.filters.plugin.version>
+        <maven.patch.filters.plugin.version>1.4.0</maven.patch.filters.plugin.version>
```
Comment (rnveach) [inline pom.xml:29]: > sevntu and patch-filters need to use a compatible/compiled version with checkstyle 

Patch filters relies on 10.4.

**Pull request merged**